.github/workflows/lint.yml

```diff
@@ -0,0 +1,57 @@
+name: lint
+
+on:
+  push:
+    branches:
+    - develop
+    - stable
+  pull_request:
+    branches:
+    - develop
+    - stable
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.8
```

Comment:
pypi shows a requiremwent
 Requires: Python >=3.8.1

maybe we should increase this here to 3.9 ?

Reply:
Depends on what minimum version of python we want to support for MSS.

Reply:
until 3.13  gets on conda it is 3.8 usually we have 5 versions

Reply:
Then it should remain 3.8 to lint with the lowest common denominator of python, I think.